index.html

```diff
@@ -5,12 +5,81 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cabin:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
-    <!-- Add your markup here -->
+     
+    <header class="header-container">
+        <img src="./images/sweet-4214168_640.png" alt="Brand logo Sweet Treats">
+        <div class="toggle-title">
+            <label for="hamburger">&#9776</label>
+            <input type="checkbox" id="hamburger">
+            <h1>The best cakes in town delivered to your door</h1>
+                <ul>
+                    <li><a href="#welcome" class="nav-link">Welcome</a></li>
+                    <li><a href="#page-images" class="nav-link">Photos</a></li>
+                    <li><a href="#contact" class="nav-link">Contact</a></li>
+                </ul>
+        </div>
+    </header>
+
+    <main class="page-wrapper">
+        <article class="welcome" id="welcome">
```

Comment:
Any particular reason you are using a class and an id here? I would use one or the other, here's a good article on when is the best time to use a class and when to use an id: https://www.developintelligence.com/blog/2016/04/css-class-vs-id-which-one-to-use/